sdk/cosmos/benchmark/pom.xml

```diff
@@ -0,0 +1,188 @@
+<!--
```

Comment:
The folder here should be `sdk/cosmos/microsoft-azure-cosmos-parent` assuming this is track 1 based on the group id being `com.microsoft.azure`. @JonathanGiles is that correct?

Reply:
fyi @mitchdenny

Reply:
will do the folder renames in the follow up PR, which will wire up the builds to central parent pom, add Emulator CI +  live tests CI.

Reply:
Just wondering, why is benchmark folder to be renamed to microsoft-azure-cosmos-parent

Reply:
Generally speaking we want the folder names containing the sources for a package to match the artifact name.